.changeset/polite-toys-happen.md

```diff
@@ -0,0 +1,7 @@
+---
+"@atproto/api": minor
+---
+
+`Agent` is no longer an abstract class. Instead it can be instantiated using object implementing a new `SessionManager` interface.
+
+
```

Comment:
We flagged this as a possible but unlikely breaking change, and if it is breaking for a project it would fail early at build/typecheck time rather than runtime.  So perhaps we can knock this down to a patch and leave a short note about what to consider if you're affected.

Reply:
```suggestion
---
"@atproto/api": patch
---

`Agent` is no longer an abstract class. Instead it can be instantiated using object implementing a new `SessionManager` interface. If your project extends `Agent` and overrides the constructor or any method implementations, consider that you may want to call them from `super`.
```